Signal/translations/en.lproj/Localizable.strings

```diff
@@ -857,6 +857,9 @@
 /* No comment provided by engineer. */
 "ERROR_MESSAGE_WRONG_TRUSTED_IDENTITY_KEY" = "Safety number changed. Tap to verify.";
 
+/* Format string for 'unregistered user' error. Embeds {{the unregistered user's signal id}}. */
+"ERROR_UNREGISTERED_USER_FORMAT" = "Unregistered User: %@";
```

Comment:
We now record which user was registered and include it in the user-facing message.

Reply:
*unregistered